package.json

```diff
@@ -63,6 +67,8 @@
     "@rubensworks/eslint-config": "^3.0.0",
     "@types/dockerode": "^3.3.29",
     "@types/jest": "^29.0.0",
+    "@types/n3": "^1.16.4",
+    "@types/streamify-array": "^1.0.3",
```

Comment:
Unused dependency?

Reply:
The N3 type is used in the code, by `streamify-array` can go.